examples/DotNetCore/WebDemo/Microsoft.Azure.AppConfiguration.WebDemo/Microsoft.Azure.AppConfiguration.WebDemo.csproj

```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
net5.0 is already out of support. Lets go with net6.0.

Reply:
I was thinking the same, but there's already a WebDemoNet6 so it seemed odd to have both

Reply:
Are there any major differences between the two? We might not need both samples. I think this one can be deleted in favor of the .NET 6 sample.

Reply:
Yes, we can delete this. If we delete this, please update the readme too
https://github.com/Azure/AppConfiguration/blob/main/examples/README.md